css/quiz.css

```diff
@@ -0,0 +1,104 @@
+body {
+	margin: 0 auto;
```

Comment:
This is not needed on the body, since there's no width on it. `margin: 0 auto` is used to center elements that are less than 100% wide on the page